src/core/thread/network_data_types.hpp

```diff
@@ -217,7 +249,7 @@ class OnMeshPrefixConfig : public otBorderRouterConfig,
 };
 
 /**
- * Represents an External Route configuration.
+ * Represents an Ip Route configuration.
```

Comment:
```suggestion
 * Represents an IP Route configuration.
```

Reply:
This seems to be an unrelated extra change (cannot recall changing it or the reason for it). I will revert it back to the original "External Route" terms.